src/Build/Resources/Strings.resx

```diff
@@ -2121,6 +2121,9 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
   <data name="TaskAssemblyLocationMismatch" xml:space="preserve">
     <value>Task assembly was loaded from '{0}' while the desired location was '{1}'.</value>
   </data>
+  <data name="ProjectImportsCollectorFileIOFail" xml:space="preserve">
+    <value>An exception occured when reading file {0}, message: {1}</value>
```

Comment:
Please wrap the inserted lines with single quotes, like:
'{0}'

@baronfel , could you please check the message content as a native speaker? :)

Reply:
occurred with two 'r' :), also maybe `when reading the file` (but not sure about it myself). Also surround `'{0}'` with single quotes